### PR TITLE
Coding Standards

### PR DESCRIPTION
**Pull request by zeroadam**
Just a little coding standards update. Didn't want to make too big, so I cut it off before Worker!

### CODE REVIEW

**Comment by zeroadam** [comment]
Here's one of those that will make @MrPetovan happy!

**Comment by MrPetovan** [comment]
Normally yes, but I’m working on those same file, dreading the merge! 😅

**Comment by zeroadam** [comment]
I can cancel the PR and wait until your next work is finished and rebase on my end to make it easier!

**Comment by MrPetovan** [comment]
Nah, I love coding standards too much for that 😛

File: mod/smilies.php [hunk 1]
```diff
@@ -6,10 +6,10 @@
 use Friendica\App;
 use Friendica\Content\Smilies;
 
-function smilies_content(App $a)
+function smiliesContent(App $a)
```
Comment (MrPetovan) [inline mod/smilies.php]: This should be kept as `smilies_content` to actually work.

Comment (zeroadam) [reply mod/smilies.php]: Ok changing back, but why will it not work? I'm sure it's something I'm not familiar with.. :smile:

**Pull request merged**